integration_tests/tests/forseti/controls/model.rb

```diff
@@ -0,0 +1,67 @@
+require 'json'
+control 'model' do
+
+    describe "Model" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+
+            # This variable cannot be used after all the inventories have been purged..
+            @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+            command("forseti model create --inventory_index_id #{@inventory_id} model_new").result
+            command("forseti model use model_new").result
+            command("sudo apt-get -y install mysql-client").result
```

Comment:
ditto: this should not be in the test, but should be rather in the setup of the VM.

Reply:
I have moved mysql-client installation at the top so it is executed only once for all the tests. I will look into moving this outside of the test file.

Reply:
Done

Reply:
You can check out the build logs here -  https://travis-ci.org/forseti-security/forseti-security/jobs/590150397